qa/rpc-tests/test_framework/util.py

```diff
@@ -340,7 +340,9 @@ def start_node(i, dirname, extra_args=None, rpchost=None, timewait=None, binary=
     datadir = os.path.join(dirname, "node"+str(i))
     if binary is None:
         binary = os.getenv("ELEMENTSD", "elementsd")
-    args = [ binary, '-chain='+chain, "-datadir="+datadir, "-server", "-keypool=1", "-discover=0", "-rest", "-mocktime="+str(get_mocktime()) ]
+    args = [ binary, "-datadir="+datadir, "-server", "-keypool=1", "-discover=0", "-rest", "-mocktime="+str(get_mocktime()) ]
+    if chain != "regtest": #TODO(stevenroose) remove if unnecessary or when bitcoind has -chain
+        args.append("-chain="+chain)
```

Comment:
why not?
```
    # Special case for regtest. Remove if bitcoind gets -chain
    if chain == 'regtest': 
        args.append('-regtest')
    else:
        args.append('-chain=' + chain)
```

Reply:
Or perhaps
```
args.append('-regtest' if chain == 'regtest' else: '-chain=' + chain)
```
or some more pythonic way to do it (haven't tried this one, but I know there's something like the '?' keyword in C, java and friends).

Reply:
Yeah I made it conditional and put a todo to remove it :)